keras_vggface/models.py

```diff
@@ -0,0 +1,526 @@
+import warnings
+
+from keras_applications.imagenet_utils import _obtain_input_shape
+from tensorflow.keras import backend as K
+from tensorflow.keras import layers
+from tensorflow.keras.layers import (Flatten,
+                                     Dense,
+                                     Input,
+                                     GlobalAveragePooling2D,
+                                     GlobalMaxPooling2D,
+                                     Activation,
+                                     Conv2D,
+                                     MaxPooling2D,
+                                     BatchNormalization,
+                                     AveragePooling2D,
+                                     Reshape,
+                                     multiply)
+from tensorflow.keras.models import Model
+from tensorflow_core.python.keras.utils import (get_file,
```

Comment:
dit doet het bij mij niet. ImportError: cannot import name 'get_file' from 'tensorflow_core.python.keras.utils'.
ik denk dat get_file uit tensorflow.python.keras.utils.data_utils moet komen? en get_source_inputs misschien uit tensorflow.keras.utils

Reply:
op de dgx werkt er nog meer niet, het volgende wel:
from tensorflow.python.keras.utils.data_utils import get_file
from tensorflow.keras.utils import get_source_inputs
from tensorflow.python.keras.utils import layer_utils

Reply:
Welke versie van tensorflow heb je? Bij mij werkt dit wel namelijk. Ik heb tensorflow 2.0.0

Reply:
2.1.0 geloof ik

Reply:
in de requirements staat tensorflow>=2.0.0 dus dan is er vast net een nieuwe uit

Reply:
Ze zijn inmiddels al bij tensorflow 2.2.X geloof ik. Ik heb de versie even gepinned op 2.0.0 voor stabiliteit